config/config.kinesis.reference.hocon

```diff
@@ -86,6 +87,7 @@
         "password": "es-pass"
         "shardDateFormat": "_yyyy-MM-dd"
         "shardDateField": "derived_tstamp"
+        "indexTimeout": 59999
```

Comment:
Out of curiosity, why `59999` and not `60000` ?

Reply:
The default in the `application.conf` file is 60000.  But the example in the reference hocon is 59999.  It's deliberately different because [we have a unit test that tests we can parse the reference hocons](https://github.com/snowplow/snowplow-elasticsearch-loader/blob/2.0.8-rc1/core/src/test/scala/com.snowplowanalytics.stream.loader/ConfigSpec.scala#L71).

I haven't decided if I like this pattern or not.  But it's what we were doing already for other parameters, so I just copied.

Reply:
I see. In enrich `application.conf` is not read at all in the unit tests (I think), but it might be different in ES loader. In that case I understand the reason.